client/packages/core/src/instaql.js

```diff
@@ -106,7 +106,9 @@ function refAttrPat(makeVar, store, etype, level, label) {
 
   const nextEtype = fwdAttr ? revEtype : fwdEtype;
 
-  return [nextEtype, nextLevel, attrPat];
+  const dir = fwdAttr ? "forward" : "reverse";
```

Comment:
nit: I would consider having this be an `isForward` variable, as the string implies an enum of more than 2.